pkg/auditd/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:edge AS mirror
```

Comment:
Ok, I set it up to build from source. We need a few of audits deps (and abuild) in the base image, so there's a commit that adds those. I think CI will fail until that's merged; I'll PR it separately assuming this approach works for people.